content/api/commands/intercept.md

```diff
@@ -364,102 +424,218 @@ cy.intercept('GET', '/should-err', { forceNetworkError: true }).as('err')
 cy.wait('@err').should('have.property', 'error')
 ```
 
-### Stubbing a response
+### routeHandler
+
+The previous examples have dealt with just spying and aliasing routes so we can wait for requests and make assertions on them in test. The remaining examples will build upon previous ones simply by specifying a `routeHandler` as the last argument to `cy.intercept`.
```

Comment:
Avoid use of the word 'simply'. We had a textlint that used to run on this repo to catch words we want to avoid/miscapitalizations, but I'm not sure it was ever set up to run again. https://github.com/cypress-io/cypress-documentation/blob/master/.textlintrc#L115:L115

Historical context: https://github.com/cypress-io/cypress-documentation/issues/2120

```suggestion
The previous examples have dealt with just spying and aliasing routes so we can wait for requests and make assertions on them in test. The remaining examples will build upon previous ones by specifying a `routeHandler` as the last argument to `cy.intercept`.
```